javascript/react/basics-i/component-lifecycle-methods.md

```diff
@@ -46,27 +47,37 @@ constructor(props) {
     time: new Date();
   }
 }
-
 ```
 
-- `componentWillMount()` - is invoked **just before** `render`. Modifying the `state` here won't trigger a **re-render**.
+- `componentDidMount()` 
```

Comment:
might be worth checking how nested lists display in the native app before merging

Reply:
cc @sourabhv

Reply:
![nested lists](https://user-images.githubusercontent.com/19802557/64259824-b770bc80-cf21-11e9-8c2c-6ee817e5d0d8.png)